src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/KeyStatisticServiceTests.cs

```diff
@@ -607,6 +765,257 @@ await using (var context = InMemoryContentDbContext(contextId))
         }
     }
 
+    [Fact]
+    public async Task UpdateKeyStatisticText()
+    {
+        var releaseId = Guid.NewGuid();
+
+        var keyStatisticText = new KeyStatisticText
+        {
+            ReleaseId = releaseId,
+            Title = "title",
+            Statistic = "statistic",
+            Trend = "trend",
+            GuidanceTitle = "guidanceTitle",
+            GuidanceText = "guidanceText",
+            Created = new DateTime(2000, 1, 1),
+            Updated = null,
+            CreatedById = Guid.NewGuid(),
+            UpdatedById = Guid.NewGuid(),
+        };
+
+        var release = new Release
+        {
+            Id = releaseId,
+            KeyStatistics = new List<KeyStatistic>
+            {
+                keyStatisticText,
+            }
+        };
+
+        var contextId = Guid.NewGuid().ToString();
+        await using (var context = InMemoryContentDbContext(contextId))
+        {
+            await context.Releases.AddRangeAsync(release);
+            await context.SaveChangesAsync();
+        }
+
+        await using (var context = InMemoryContentDbContext(contextId))
+        {
+            var keyStatisticService = SetupKeyStatisticService(context);
+            var result = await keyStatisticService.UpdateKeyStatisticText(
+                release.Id,
+                keyStatisticText.Id,
+                new KeyStatisticTextUpdateRequest
+                {
+                    Title = "new title",
+                    Statistic = "new statistic",
+                    Trend = "new trend",
+                    GuidanceTitle = "new guidanceTitle",
+                    GuidanceText = "new guidanceText",
+                });
+
+            var viewModel = result.AssertRight();
+
+            Assert.Equal("new title", viewModel.Title);
+            Assert.Equal("new statistic", viewModel.Statistic);
+            Assert.Equal("new trend", viewModel.Trend);
+            Assert.Equal("new guidanceTitle", viewModel.GuidanceTitle);
+            Assert.Equal("new guidanceText", viewModel.GuidanceText);
+            Assert.Equal(0, viewModel.Order);
+            Assert.Equal(new DateTime(2000, 1, 1), viewModel.Created);
+            Assert.NotNull(viewModel.Updated);
+            Assert.InRange(DateTime.UtcNow.Subtract(viewModel.Updated!.Value).Milliseconds, 0, 1500);
+        }
+
+        await using (var context = InMemoryContentDbContext(contextId))
+        {
+            var keyStatistics = context.KeyStatistics.ToList();
+            var keyStat = Assert.Single(keyStatistics);
```

Comment:
Do we need to assign `keyStat` to a variable here?

Reply:
I think this is super minor, to the point where it really doesn't matter.

I think I prefer the way it is: if I remove this variable we'll have this code instead:
```
            Assert.Single(keyStatistics);
            var keyStatText = Assert.IsType<KeyStatisticText>(keyStatistics[0]);
```
I'd prefer to avoid the `keyStatistics[0]`. I also like having the variable nams: `keyStatText` hints that the type has been checked, where `keyStat` implies it hasn't.

I can see that you might want to avoid having a `keyStat` and `keyStatText` variable. But, as I say, this change seems  practically inconsequential to me.